ddtrace/tracer/option.go

```diff
@@ -446,6 +453,9 @@ func newConfig(opts ...StartOption) *config {
 		}
 		c.dogstatsdAddr = addr
 	}
+	if c.debugOpenSpans {
+		c.spanTimeout = internal.DurationEnv("DD_TRACE_OPEN_SPAN_TIMEOUT", time.Second)
```

Comment:
I'm not so sure about this 1 second default. Can you give some context about your decision for that?
For a root span, it wouldn't be unheard for a span to be open for a while during the processing of children operations. Let's go with something that we know will definitely be too long, like 10 minutes or something like that.

Reply:
Makes sense! I wasn't entirely sure what the best timeout would be to go for. Something longer like 5-10 minutes sounds good.